install/update_95_xx.php

```diff
@@ -94,6 +94,47 @@ function update95toXX() {
       }
    }
 
+   // Recurrent changes
+   $DB->updateOrDie(CronTask::getTable(), [
+         'itemtype' => CommonITILRecurrentCron::class,
```

Comment:
I think using hard coded strings is better in migration, to prevent migration to be altered if we refactor the code.
```suggestion
         'itemtype' => 'CommonITILRecurrentCron',
```